src/eureka/S3_data_reduction/optspex.py

```diff
@@ -28,8 +30,18 @@ def standard_spectrum(data, apdata, aperr):
     data : Xarray Dataset
         The updated Dataset object in which the spectrum data will stored.
     """
-    data['stdspec'] = (['time', 'x'], np.sum(apdata, axis=1))
-    data['stdvar'] = (['time', 'x'], np.sum(aperr ** 2, axis=1))
+    # Get numbers to account for missing pixels
+    npix = apmask.shape[1]
+    nmasked = np.sum(apmask == 0, axis=1)
+    correction = npix/(npix-nmasked)
```

Comment:
I'm not keen on this correction strategy. The flux isn't uniform down a pixel column. It's really concentrated into 2-3 pixels. Thus, there's a 50/50 chance of under/over correcting and no chance getting it right. Ultimately, the optimal spectral extraction will get it right.

I don't have a helpful suggestion at the moment. Happy to chat.

Reply:
Maybe we can replace the bad pixel with an average of its neighbors on the left and right.

Reply:
While I agree that this is certainly not ideal, it was a small step in the right direction compared to not doing any correction based on the testing I did. I also agree that the optimal extraction ultimately figures things out all the same - this was just a beginning attempt at patching the standard spectra a bit since several people have asked me if they're able to use the standard spectra instead of the optimal spectra (for various reasons, e.g. easier aperture correction for flux calibrated spectra).

Reply:
That's not a bad suggestion either, although it comes with complications if there are adjacent masked pixels

Reply:
My most recent commit implements your suggestion (using two neighbors on either side since adjacent bad pixels are fairly likely) and looks like it works really quite well.

Here is the stdspec and optspec for a GTO MIRI observation that I got with my previous hacky method
![fig3302_file00_int0000_Spectrum](https://github.com/kevin218/Eureka/assets/24514122/28833819-5aa1-443e-a99e-0c27f7abb13f)

while here is the stdspec and optspec of the same data with the new algorithm
![fig3302_file00_int0000_Spectrum](https://github.com/kevin218/Eureka/assets/24514122/5b4789bf-84d1-43a1-baf6-ddbe6e5aeed8)

The stdspec is now looking *much* better and is actually quite usable (for those that want it)